icicle_v3/backend/cpu/include/ntt_template.h

```diff
@@ -0,0 +1,264 @@
+#include "icicle/ntt.h"
```

Comment:
I know that I named the file ntt.template but maybe cpu_ntt.h is better to be more consistent with the code base which is full of templates but not named with 'template' in the file name.